<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchLotDialog</class>
 <widget class="QDialog" name="SearchLotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonTitleLotID">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSubCode">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitleSubCode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTitleLotNumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Lot number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLotNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLotID">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSelect">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>選択</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
